DigitalLearningSolutions.Web/Views/Shared/MiniHub/_ContentsList.cshtml

```diff
@@ -4,16 +4,19 @@
 <nav class="nhsuk-contents-list" role="navigation" aria-label="Pages in this guide">
   <h2 class="nhsuk-u-visually-hidden">Contents</h2>
   <ol class="nhsuk-contents-list__list">
-    @for (var pageIndex = 0; pageIndex < @Model.Sections.Count; pageIndex++) {
-      <li class="nhsuk-contents-list__item" aria-current="page">
-        @{ var sectionToRender = Model.Sections[pageIndex]; }
-        @if (Model.CurrentSectionIndex == pageIndex) {
-          <span class="nhsuk-contents-list__current">@sectionToRender.SectionTitle</span>
+    @for (var sectionIndex = 0; sectionIndex < @Model.Sections.Count; sectionIndex++) {
+
+      var isCurrentSection = Model.CurrentSectionIndex == sectionIndex;
+      var sectionToLinkTo = Model.Sections[sectionIndex];
+
+      <li class="nhsuk-contents-list__item" @Html.Raw(isCurrentSection ? @"aria-current=""page""" : string.Empty)>

```

Comment:
You should be able to do something like aria-current=@(isCurrentSection ? "page" : null) and Razor will exclude the aria-current attribute by default.